drivers/classic/src/initialize_atmos.c

```diff
@@ -1013,17 +971,16 @@ initialize_atmos(atmos_data_struct    *atmos,
     if (param_set.TYPE[SHORTWAVE].SUPPLIED) {
         have_shortwave = 1; // flag for MTCLIM
         for (day = 0; day < Ndays_local; day++) {
-            for (hour = 0; hour < HOURS_PER_DAY; hour++) {
+            for (step = 0; step < atmos_steps_per_day; step++) {
                 if (param_set.FORCE_DT[param_set.TYPE[SHORTWAVE].SUPPLIED -
-                                       1] == HOURS_PER_DAY) {
-                    hourlyrad[day * HOURS_PER_DAY +
-                              hour] = local_forcing_data[SHORTWAVE][day];
+                                       1] == SEC_PER_DAY) {
+                    subdailyrad[day * SEC_PER_DAY +
+                                step] = local_forcing_data[SHORTWAVE][day];
```

Comment:
Don't break in the middle of the `[]` - even if we did so before. Check the rest of this file as well as that seems to happen a lot, even if we have less than 80 characters per line